int8 kind
int8 slot_id
ItemInfo info
#geometry_msgs/PoseStamped pose
